src/common/xio_connection.c

```diff
@@ -3341,3 +3352,35 @@ void xio_connection_keepalive_start(void *_connection)
 	}
 }
 
+/*---------------------------------------------------------------------------*/
+/* xio_connection_force_disconnect					     */
+/*---------------------------------------------------------------------------*/
+int xio_connection_force_disconnect(struct xio_connection *connection,
```

Comment:
Please fix spacing (tabs, etc...) in this function according to coding conventions.